src/com/facebook/buck/rules/coercer/CollectionTypeCoercer.java

```diff
@@ -51,7 +51,7 @@ protected void fill(
       CellPathResolver cellRoots,
       ProjectFilesystem filesystem,
       Path pathRelativeToProjectRoot,
-      C.Builder<T> builder,
+      ImmutableCollection.Builder<T> builder,
```

Comment:
remember that java doesn't care about generic type at compile time anyway, this is totally safe.